code/UI/GameStatePanel/GameStatePanel.cs

```diff
@@ -11,4 +11,16 @@ public static void StartGame()
 		if ( Game.Clients.Count >= GrubsConfig.MinimumPlayers && ffa.TerrainReady && !ffa.Started )
 			ffa.Start();
 	}
+
+	[ConCmd.Server]
+	public static void RestartGame()
+	{
+		Game.ResetMap( Array.Empty<Entity>() );
+
+		GamemodeSystem.Instance.Delete();
+		GamemodeSystem.SetupGamemode();
+		GamemodeSystem.Instance.GameWorld = new World();
```

Comment:
Seems a bit crappy to have to create it, should `SetupGamemode` maybe handle it? Or do we not care for now...